whitepaper/chapters/accounts.tex

```diff
@@ -13,9 +13,9 @@ \subsection{Addresses}
 
 An \nind{address} is a base32\footnote{ \url{http://en.wikipedia.org/wiki/Base32} } encoded triplet consisting of:
 \begin{itemize}
-	\item{network byte}
-	\item{160-bit hash of the account's public key}
-	\item{4 byte checksum}
+	\item{network byte.}
+	\item{160-bit hash of the account's public key.}
+	\item{4 byte checksum.}
```

Comment:
if you're adding periods, should you capitalize first word?

Reply:
In the latest commit, bullet lists with uncomplete sentences are written without the period.